src/tuplet.ts

```diff
@@ -81,7 +110,7 @@ export class Tuplet extends Element {
     if ('bracketed' in this.options) {
       this.bracketed = this.options.bracketed;
     } else {
-      this.bracketed = notes.some((note) => note.beam === undefined);
+      this.bracketed = notes.some((note) => note.getBeam() === undefined);
```

Comment:
You can use `hasBeam` as mentioned above.

Reply:
done